src/vm/dllimport.cpp

```diff
@@ -5791,8 +5809,14 @@ class LoadLibErrorTracker
         }
     }
 
+    void UpdateHRUnix(LPCWSTR message)
```

Comment:
This is updating message, not HR. You can call this SetMessage; and keep the name of UpdateHR as is.